src/lucky_router/match_finder.cr

```diff
@@ -20,20 +21,19 @@ class LuckyRouter::MatchFinder(T)
   # This uses the magic/pain of recursion to continue matching fragments
```

Comment:
These comments no long make sense since this doesn't use the pain of recursion! 😂